Models/NewCard.cs

```diff
@@ -131,35 +134,28 @@ public static class NewCard
 				card.titleGraphic = titleGraphic;
 			}
 
-			NewCard.cards.Add(card);
-
-			handleIdentifiers(card, abilityIds, evolveId, iceCubeId, tailId);
-
-			Plugin.Log.LogInfo($"Loaded custom card {name}!");
+			NewCard.Add(card, abilityIdsParam, evolveId, iceCubeId, tailId);
 		}
 
-		private static void handleIdentifiers(CardInfo card, List<AbilityIdentifier> abilityIds, EvolveIdentifier evolveId,
-		IceCubeIdentifier iceCubeId, TailIdentifier tailId)
+		private static void handleIdentifiers(
+			List<AbilityIdentifier> abilityIdsParam = null, EvolveIdentifier evolveId = null, 
+			IceCubeIdentifier iceCubeId = null, TailIdentifier tailId = null
+		)
 		{
-			// Handle AbilityIdentifier
 			List<AbilityIdentifier> toRemove = new List<AbilityIdentifier>();
-			if (abilityIds is not null)
+			if (abilityIdsParam is not null)
 			{
-				foreach (AbilityIdentifier id in abilityIds)
-				{
-					if (id.id != 0)
-					{
-						card.abilities.Add(id.id);
```

Comment:
This is removed since IdentifierHandler will add the ID to the card. If this is kept it, this will add the ability twice.